src/webui/www/private/preferences_content.html

```diff
@@ -1099,6 +1356,54 @@
         return ret;
     };
 
+    // Advanced Tab
+    var updateNetworkInterfaces = function(default_iface) {
```

Comment:
you are free to use `const` & `let` instead of `var`, please use them in new code.